src/tests/sbus_codegen_tests.c

```diff
@@ -115,15 +115,15 @@ END_TEST
 
 START_TEST(test_signals)
 {
-    const struct sbus_signal_meta *signal;
+    const struct sbus_signal_meta *sm_signal;
```

Comment:
It is not `sbus_meta_signal` but `sbus_signal_meta` But `ssm_` does not increase readability either

What about `a_signal` or `sig`, `signl`